pycloudlib/tests/lxd/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests related to pycloudlib.lxd module."""
```

Comment:
I would suggest we align with other projects and have a tests subdirectory below the module we are testing (not a top-level pycloudlib/tests  directory that has a full mapping of all submodules.

I think the lxd tests should live under `pycloublib/lxd/tests` instead of `pycloudlib/tests/lxd` so devs don't have to cd  up and across from a top-level directory.

```